state.go

```diff
@@ -117,16 +118,17 @@ func (a *Autopilot) gatherNextStateInputs(ctx context.Context) (*nextStateInputs
 				break
 			}
 		}
-		if inputs.LeaderID == "" {
```

Comment:
This is no longer a failure condition as we allow computing the state even when there is no leader (we will later on mark everything as unhealthy though)